backend/src/services/linear/api/resolvers.py

```
@@ -10351,9 +10351,9 @@ def resolve_issueAddLabel(obj, info, **kwargs):
         # Add the label to the issue
         issue.labels.append(label)
 
-        # Update the labelIds list
+        # Update the labelIds list (must reassign to trigger SQLAlchemy change tracking for JSONB)
         if label_id not in issue.labelIds:
-            issue.labelIds.append(label_id)
+            issue.labelIds = issue.labelIds + [label_id]
```

Ruff (0.14.4)

Consider `[*issue.labelIds, label_id]` instead of concatenation

Replace with `[*issue.labelIds, label_id]`

(RUF005)